app/src/main/java/com/example/weather/ui/maps/MapsViewModel.java

```diff
@@ -5,51 +5,42 @@
 import android.content.Context;
 import android.content.Intent;
 import android.util.Log;
-import android.view.View;
-import android.widget.Toast;
 
 import androidx.annotation.NonNull;
 import androidx.lifecycle.AndroidViewModel;
 import androidx.lifecycle.ViewModel;
+import androidx.room.Database;
 
-import com.example.weather.data.db.entity.ForecastData;
+import com.example.weather.data.db.database.WeatherDatabase;
+import com.example.weather.data.db.entity.WeatherEntity;
 import com.example.weather.data.db.entity.ForecastDetail.Coord;
 import com.example.weather.data.network.api.ForecastApi;
-import com.example.weather.ui.Forecast.ForecastActivity;
 import com.example.weather.utils.Constant;
 import com.google.android.gms.maps.model.LatLng;
 
 import io.reactivex.android.schedulers.AndroidSchedulers;
 import io.reactivex.functions.Consumer;
 import io.reactivex.schedulers.Schedulers;
 
-public class MapsViewModel extends ViewModel {
+public class MapsViewModel extends AndroidViewModel {
```

Comment:
`AndroidViewModel` is deprecated. Why do you remove `ViewModel`?

Reply:
Because I need context